provider/awssd/aws_sd_test.go

```diff
@@ -39,6 +39,10 @@ import (
 // Compile time check for interface conformance
 var _ AWSSDClient = &AWSSDClientStub{}
 
+var (
+	errNamespaceNotFound = errors.New("Namespace not found")
```

Comment:
```suggestion
	ErrNamespaceNotFound = errors.New("Namespace not found")
```

Otherwise it can't be tested for.